detailed_docs/RC/resource_allocation.md

```diff
@@ -150,32 +187,37 @@ _Steps:_
 
 1. User disables RC functionality
 2. HMI sent notification to SDL that RC functionality is disabled
-3. SDL assigns HMILevel NONE to RC_app_1 and RC_app_2 and unsubscribes applications from module_1
+3. SDL assigns HMILevel NONE to RC_app_1 and RC_app_2 and unsubscribes applications from module_1  
+4. SDL sends OnRCStatus notification to RC_app_1 and RC_app_2 and to HMI with module_1 in `freeModules` parameter
 4. User activates RC functionality with "accessMode" = "ASK_DRIVER"
 5. User activates RC_app_2, module_1 gets in "free" state again
 6. RC_app_2 requests the control of module_1
 
 _Expected:_
 
-7. RC_app_2 gets in control of module_1 without asking a driver
-8. User activates RC_app_1 and RC_app_1 requests the control of module_1
-9. SDL sends the request to HMI to display permission prompt to the driver
+7. RC_app_2 gets in control of module_1 without asking a driver  
+8. SDL sends OnRCStatus to RC_app_1 and RC_app_2 and to HMI with module_1 in `allocatedModules` parameter.
```

Comment:
module_1 can only be in one allocatedModules

Reply:
@yang1070, thank you. 
please see updates https://github.com/smartdevicelink/sdl_requirements/pull/89/commits/bcefffcf4992deb88e3ab84717ac519cbd318197